docs/src/install.md

```diff
@@ -6,6 +6,32 @@ to install the latest stable version of this package:
 ```julia
 pkg> add Bloqade
 ```
+## Using Bloqade Optimized AMIs from AWS Marketplace
+The Bloqade team has implemented 2 dedicated AMIs that can be acquired using AWS Marketplace
+
+- Bloqade AMI with Julia
+- Bloqade Optimized Deep Learning AMI with CUDA and Julia
```

Comment:
Maybe we can just say CUDA? There is nothing to do with deep learning for us IIUC

Reply:
yes sir